bazel/dependency_imports.bzl

```diff
@@ -15,7 +15,7 @@ def _pgv_pip_dependencies():
 def _pgv_go_dependencies():
     go_rules_dependencies()
     go_register_toolchains(
-        version = "1.15.6",
+        version = "1.18.5",
```

Comment:
Not 1.19?

Reply:
arbitrary "one behind is safe" - happy to bump to 19 if you are...

Reply:
Yep, let's do it.

Reply:
resolved in c3c2b4e